pyphare/pyphare/pharein/examples/job.py

```diff
@@ -1,25 +1,20 @@
 #!/usr/bin/env python
 
-
-from pyphare.pharein import Simulation
-from pyphare.pharein import MaxwellianFluidModel
-from pyphare.pharein import ElectronModel
-from pyphare.pharein import ElectromagDiagnostics
-from pyphare.pharein import FluidDiagnostics
-from pyphare.pharein import getSimulation
+import numpy as np
+import pyphare.pharein as ph
 
 
 # ------------------------------------
 #     configure the simulation
 # ------------------------------------
 
-Simulation(
+ph.Simulation(
     time_step_nbr=1000,  # number of time steps (not specified if time_step and final_time provided)
     final_time=1.0,  # simulation final time (not specified if time_step and time_step_nbr given)
     boundary_types="periodic",  # boundary condition, string or tuple, length == len(cell) == len(dl)
     cells=80,  # integer or tuple length == dimension
     dl=0.1,  # mesh size of the root level, float or tuple
-    path="test5"  # directory where INI file and diagnostics directories will be
+    path="test5",  # directory where INI file and diagnostics directories will be
```

Comment:
I don't think this is used anymore (not really part of this PR tho)

Reply:
yeah we should put an issue to care about that

Reply:
done the issue